test/suites/fips/validate-fips.robot

```diff
@@ -31,17 +31,29 @@ Setup
     [Documentation]    Test suite setup
     Check Required Env Variables
     Login MicroShift Host
+    Setup Kubeconfig
+    Wait Until Greenboot Health Check Exited
+    Stop MicroShift
 
 Teardown
     [Documentation]    Test suite teardown
+    Start MicroShift
```

Comment:
This is a bit assymetric because we're stopping MicroShift in a test, while starting it in the suite Teardown.
Do you think we should shut down the service in Setup?